README.md

```diff
@@ -41,7 +41,10 @@ virtual machine:
  * Run `vagrant plugin install vagrant-registration` to install plugin
 * [vagrant-adbinfo plugin](https://github.com/bexelbie/vagrant-adbinfo) _(>=0.0.9)_ installed
  * Run `vagrant plugin install vagrant-adbinfo` to install plugin
-
+* On Windows:
+ * Ensure putty utilities, including pscp, are installed and on the Path
```

Comment:
See https://github.com/projectatomic/vagrant-adbinfo/issues/20

Currently the plugin requires pscp, not just any SCP